packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -234,7 +234,7 @@ export default class Editor {
     }
 
     // Insert content into editor
-    public insertContent(content: string, option: InsertOption): void {
+    public insertContent(content: string, option?: InsertOption): void {
```

Comment:
Why do you not set a default 'option' in this function? Could option be null?

Reply:
I can, but then the code will be duplicated in insertNode(). So I just set it once there.

Reply:
Oh I didn't see it calls insertNode() eventually.